app/terms/page.tsx

```
@@ -71,332 +133,378 @@ export default function TermsOfServicePage() {
+            {/* Main Content */}
+            <div className="flex-1 max-w-3xl">
+              <div className="rounded-3xl border border-white/70 bg-white/80 shadow-xl shadow-violet-950/10 backdrop-blur-xl dark:border-white/10 dark:bg-white/[0.04] dark:shadow-black/20">
+                <div className="px-6 py-10 sm:px-10 md:px-14 md:py-14">
+                  <div className="prose prose-zinc max-w-none dark:prose-invert prose-headings:scroll-mt-24 prose-p:leading-8 prose-li:leading-7">
+                    {/* Section 1 */}
+                    <section id="introduction" className={sectionClass}>
+                      <h2 className="text-3xl font-bold tracking-tight text-zinc-900 dark:text-white">
+                        1. Introduction
+                      </h2>
+                      <p>
+                        Welcome to <strong>LinkID</strong> ("we," "our," or
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&`#34`;`, `&rdquo;`.

(react/no-unescaped-entities)